src/hooks/directoryHooks/useUpdateDirectoryHook.jsx

```diff
@@ -42,10 +42,7 @@ export function useUpdateDirectoryHook(params, queryParams) {
             _.omit(params, "collectionName"),
           ])
         else if (params.resourceCategoryName)
-          queryClient.invalidateQueries([
-            DIR_CONTENT_KEY,
-            _.omit(params, "resourceCategoryName"),
-          ])
+          queryClient.invalidateQueries([RESOURCE_ROOM_CONTENT_KEY])
```

Comment:
We do still need to include `_.omit(params, "resourceCategoryName")`

Reply:
double checking here - this is to have a more precise invalidation strategy for our query client right?

Reply:
Yep - using `invalidateQueries([RESOURCE_ROOM_CONTENT_KEY])` would also invalidate all resource room content on other sites visited by a user, so I think it would be good to invalidate only the specific item that has changed here

Reply:
i **think** how it works is that the above invalidation will also result in `useGetResourceRoom` triggering a refetch as the query key is more specific (array of length 2 - `[RESOURCE_ROOM_CONTENT_KEY, params]`). 

we also do **not** need to do an omit as `resourceCategoryName` doesn't exist on the param - the type is as follows
```ts 
interface ResourceRoomRouteParams {
  siteName: string
  resourceRoomName: string
}
`
``

Reply:
Ah right, but we should still include the params `queryClient.invalidateQueries([RESOURCE_ROOM_CONTENT_KEY, params])`, since we don't want to invalidate the resource room content of other sites

Reply:
oh yea, i missed that - my bad and you're totally right